From 8261c23332ce89605d8e3b7a357a8f39e3fb0cbe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 23 Oct 2025 19:26:42 +0200
Subject: [PATCH] Fix port to Empy-4.2

The PR #123 replaced the $ sign with @, but it caused that the values were
not substitued, which procedured an invalid value. Revert it, and add a
space between brackets to fix compilation with Empy-4.2.
---
 gtk/theme/gtkrc.em                 | 4 ++--
 gtk3/theme/3.20/gtk-widgets.css.em | 2 +-
 gtk3/theme/gtk-widgets.css.em      | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/gtk/theme/gtkrc.em b/gtk/theme/gtkrc.em
index 6343747..7586be7 100644
--- a/gtk/theme/gtkrc.em
+++ b/gtk/theme/gtkrc.em
@@ -691,7 +691,7 @@ style "menuitem"
     text[ACTIVE]      = $white
 
     xthickness = $subcell_size
-    ythickness = @((subcell_size * 3 - font_height) / 2)
+    ythickness = $( (subcell_size * 3 - font_height) / 2)
 }
 
 style "checkmenuitem"
@@ -699,7 +699,7 @@ style "checkmenuitem"
     GtkCheckMenuItem::indicator-size = $radio_size
     GtkMenuItem::toggle-spacing = $(subcell_size * 2 / 3)
 
-    ythickness = @((subcell_size * 3 - max(font_height, subcell_size * 2 / 3)) / 2)
+    ythickness = $( (subcell_size * 3 - max(font_height, subcell_size * 2 / 3)) / 2)
 
     # This is only there because of bug #382646 ...
     base[NORMAL]      = $white
diff --git a/gtk3/theme/3.20/gtk-widgets.css.em b/gtk3/theme/3.20/gtk-widgets.css.em
index 0b953a5..3101f0d 100644
--- a/gtk3/theme/3.20/gtk-widgets.css.em
+++ b/gtk3/theme/3.20/gtk-widgets.css.em
@@ -423,7 +423,7 @@ menu {
 }
 
 menuitem {
-    padding: $(subcell_size)px @((subcell_size * 3 - font_height) / 2)px;
+    padding: $(subcell_size)px $( (subcell_size * 3 - font_height) / 2)px;
 }
 
 menuitem:hover,
diff --git a/gtk3/theme/gtk-widgets.css.em b/gtk3/theme/gtk-widgets.css.em
index 8411bfd..a1eb9e8 100644
--- a/gtk3/theme/gtk-widgets.css.em
+++ b/gtk3/theme/gtk-widgets.css.em
@@ -489,7 +489,7 @@ SugarPaletteWindow SugarGroupBox *:insensitive {
 }
 
 .menuitem {
-    padding: $(subcell_size)px @((subcell_size * 3 - font_height) / 2)px;
+    padding: $(subcell_size)px $( (subcell_size * 3 - font_height) / 2)px;
 }
 
 .menuitem:prelight, .menuitem:hover {
